### DISCUSSION TITLE
[projects beta] Appearance on Dark Color blind them is broken

### DISCUSSION THREAD

**Discussion by WillsterJohnson** [community/community#6851] [Projects and Issues] [answered]
## What's the problem?

I'm listing this under "[projects beta]" because this is the only place where I've seen theme issues with the Dark colorblind theme.
I've also gona ahead and checked the appearance on the other 5 themes and they all seem correct, so my best guess is theres a seldom used token in the CSS that's gone wrong (probably the `aHKhM` class, whatever that means).

When using the "Dark colorblind" theme option, some of the major UI elements are white with some text being low contrast.
The table layout is completely white, and the board layout is partially white.

If this is intentional, then take this as a request for a change, though I assume its an accident in the CSS somewhere.

## Screenshots

![Example of Dark Colorblind theme causing color issues on projects beta - table view](https://user-images.githubusercontent.com/85991558/139148399-500ac28f-2508-4327-8a34-18b092024edb.png)
*Example of Dark Colorblind theme causing color issues on projects beta - table view*

![Example of Dark Colorblind theme causing color issues on projects beta - board view](https://user-images.githubusercontent.com/85991558/139148006-3afb05ef-a573-41a9-9ca2-a05173b0c646.png)
*Example of Dark Colorblind theme causing color issues on projects beta - board view*

I really hope this is as simple as swapping out a class or token and pushing a bug fix, because I'd like to try out this beta feature but I don't want to have to switch themes every time I use it.

## It seems that this was [raised almost 2 weeks ago](https://github.com/github/feedback/discussions/6363) and nobody responded, and then [raised again shortly after](https://github.com/github/feedback/discussions/6455) to the same silence

**Comment by mariorod** [comment, marked as answer]
Hi, we are working on these and we expect to get a fix to production in the coming week.

**Reply by WillsterJohnson** [reply]
That's great to hear! Thanks for getting on this
